### ISSUE TITLE
daemonization

### ISSUE THREAD

**Issue by insidenothing** [opentimestamps/opentimestamps-server#80]
"To actually run the server, run the otsd program. Proper daemonization isn't implemented yet, so otsd runs in the foreground. "

Is it worth noting using screen, solves issues with terminal disconnect and background tho not a daemon?
<pre>
screen -S opentimestampserver
./otsd
</pre>
* Detach: Ctrl+a d 
* Find: screen ls
* Reattach: screen -r

**Comment by cisba** [comment]
TLDR: 
With systemd, a service can now be written as if it was running in the foreground, just like any other program. So probably you can use the service configuration I wrote [here](https://github.com/cisba/opentimestamps-rpm/tree/master/server).

Long answer:
The old style python daemons following [PEP 3143](https://www.python.org/dev/peps/pep-3143/) is provided by the [python-daemon](https://pypi.org/project/python-daemon/) standard library.

(BTW: When I tried to use it for a small script I ran into some problems and I found mixed opinions around.)

Systemd allow a [new-style daemons in Python](https://www.loggly.com/blog/new-style-daemons-python/): simply don't daemonize your code at all!

If you don't have Systemd you can try [supervisord](http://supervisord.org/introduction.html).

A nice post with many citation is [this](https://dpbl.wordpress.com/2017/02/12/a-tutorial-on-python-daemon/) too.

**Comment by insidenothing** [comment]
Thank you, worked perfectly!

[ec2-user@ip-172-31-61-7 systemd]$ sudo service otsd.service status
Redirecting to /bin/systemctl status otsd.service
● otsd.service - OpenTimestamps server
   Loaded: loaded (/etc/systemd/system/otsd.service; disabled; vendor preset: disabled)
   Active: active (running) since Tue 2021-02-02 17:41:34 UTC; 11s ago
 Main PID: 28057 (python3)
   CGroup: /system.slice/otsd.service
           └─28057 python3 /home/ec2-user/opentimestamps-server/otsd

Feb 02 17:41:34 ip-172-31-61-7.ec2.internal systemd[1]: Started OpenTimestamps server.
Feb 02 17:41:34 ip-172-31-61-7.ec2.internal systemd[1]: Starting OpenTimestamps server...
Feb 02 17:41:35 ip-172-31-61-7.ec2.internal otsd[28057]: Opening journal for appending...
Feb 02 17:41:35 ip-172-31-61-7.ec2.internal otsd[28057]: Journal has 1 entries
Feb 02 17:41:35 ip-172-31-61-7.ec2.internal otsd[28057]: Starting aggregator loop
Feb 02 17:41:35 ip-172-31-61-7.ec2.internal otsd[28057]: Starting stamper loop
